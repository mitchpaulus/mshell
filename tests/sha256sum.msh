`sha256sum.msh` sha256sum wl
