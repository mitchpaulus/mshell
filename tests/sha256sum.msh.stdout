963bb88d79ec8b094337a137a6823437c1c0dd807befa3a719e8cf398f0de1f8
